### PR TITLE
add fmv to snapshot and ws models and add new feeds

### PR DESCRIPTION
**Pull request by morningvera**

### CODE REVIEW

File: polygon/websocket/models/common.py [hunk 1]
```diff
@@ -9,6 +9,16 @@ class Feed(Enum):
...
     Launchpad = "launchpad.polygon.io"
+    BusinessFeed = "business.polygon.io"
+    EdgxBusinessFeed = "edgx-business.polygon.io"
+    DelayedBusinessFeed = "delayed-business.polygon.io"
+    DelayedEdgxBusinessFeed = "delayed-edgx-business.polygon.io"
+    DelayedNasdaqLastSaleBusinessFeed = "delayed-nasdaq-last-sale-business.polygon.io"
+    DelayedNasdaqBasicFeed = "delayed-nasdaq-basic-business.polygon.io"
+    DelayedFullMarketBusinessFeed = "delayed-fullmarket-business.polygon.io"
+    FullMarketBusinessFeed = "fullmarket-business.polygon.io"
+    NasdaqfeedLastSaleBusinessFeed = "nasdaq-last-sale-business.polygon.io"
```
Comment (aitzkovitz) [inline polygon/websocket/models/common.py:20]: we can probably omit _feed_ from these variable names to stay consistent with the delayed names

Comment (morningvera) [reply polygon/websocket/models/common.py:20]: this was just copy/paste from the go client, good point 👍

File: test_rest/mocks/v2/snapshot/locale/us/markets/stocks/tickers/AAPL.json [hunk 2]
```diff
@@ -47,6 +47,7 @@
         "ticker": "AAPL",
         "todaysChange": -3.65,
         "todaysChangePerc": -2.231,
-        "updated": 1651251948294080343
+        "updated": 1651251948294080343,
+        "fair_market_value": 160.315
```
Comment (aitzkovitz) [inline test_rest/mocks/v2/snapshot/locale/us/markets/stocks/tickers/AAPL.json:51]: shouldn't this be "fmv", not "fair_market_value"

Comment (morningvera) [reply test_rest/mocks/v2/snapshot/locale/us/markets/stocks/tickers/AAPL.json:51]: yep, good catch

File: polygon/websocket/models/common.py [hunk 3]
```diff
@@ -9,6 +9,16 @@ class Feed(Enum):
...
+    Business = "business.polygon.io"
+    EdgxBusiness = "edgx-business.polygon.io"
+    DelayedBusiness = "delayed-business.polygon.io"
+    DelayedEdgxBusiness = "delayed-edgx-business.polygon.io"
+    DelayedNasdaqLastSaleBusiness = "delayed-nasdaq-last-sale-business.polygon.io"
+    DelayedNasdaqBasic = "delayed-nasdaq-basic-business.polygon.io"
+    DelayedFullMarketBusiness = "delayed-fullmarket-business.polygon.io"
+    FullMarketBusiness = "fullmarket-business.polygon.io"
+    NasdaqfeedLastSaleBusiness = "nasdaq-last-sale-business.polygon.io"
+    NasdaqfeedBasicBusiness = "nasdaq-basic-business.polygon.io"
```
Comment (aliche-a) [inline polygon/websocket/models/common.py:21]: should we rename these variables to be `NasdaqLastSaleBusiness` and `NasdaqBasicBusiness` for consistency with the endpoints?

Comment (morningvera) [reply polygon/websocket/models/common.py]: done!

File: polygon/websocket/models/common.py (hunk 1 above)
Comment (aitzkovitz) [reply polygon/websocket/models/common.py]: oh wait, my comment here was unclear. I think we should still append "Feed" to every one of these. 

But i think we should replace "Nasdaqfeed" with just "Nasdaq" for the nasdaq feeds

Comment (morningvera) [reply polygon/websocket/models/common.py]: why? none of the original feed names include `Feed` other than `Polyfeed(Plus)` and `StarterFeed` which is just part of the product name in those cases

Comment (aitzkovitz) [reply polygon/websocket/models/common.py]: Ah okay, I thought they all had Feed at the end, nevermind about that then. 

But I still think it makes sense to replace "Nasdaqfeed" with just "Nasdaq" for the nasdaq feeds

Comment (morningvera) [reply polygon/websocket/models/common.py]: already done, see alice's comment

**Pull request merged**